package.json

```diff
@@ -137,7 +137,8 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.1.1",
     "request-promise-native": "^1.0.8",
-    "sqlite3": "^4.1.1",
+    "sqlite": "^4.0.14",
```

Comment:
Things that get added to the package are used for my tile generation process, I'm figuring out how to add that as a PR since it has a lot of large source files and I don't want to bloat the repo